docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
+.. _punctuation_and_capitalization:
+
+Punctuation and Capitalization Model
+====================================
+
+Automatic Speech Recognition (ASR) systems typically generate text with no punctuation and capitalization of the words. \
+There are two issues with non-punctuated ASR output:
+
+- it could be difficult to read and understand;
+- models for some downstream tasks such as named entity recognition, machine translation or text-to-speech are usually trained on punctuated datasets and using raw ASR output as the input to these models could deteriorate their performance.
+
+Model Description
+-----------------
+
+For each word in the input text, the Punctuation and Capitalization model:
+
+1. predicts a punctuation mark that should follow the word (if any). By default, the model supports commas, periods and question marks.
+2. predicts if the word should be capitalized or not.
+
+.. note::
+
+    We recommend you try this model in a Jupyter notebook \
+    (can run on `Google's Colab <https://colab.research.google.com/notebooks/intro.ipynb>`_.): \
+    `NeMo/tutorials/nlp/Punctuation_and_Capitalization.ipynb <https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Punctuation_and_Capitalization.ipynb>`__.
+
+    Connect to an instance with a GPU (Runtime -> Change runtime type -> select "GPU" for hardware accelerator)
+
+    An example script on how to train the model could be found here: `NeMo/examples/nlp/token_classification/punctuation_capitalization_train.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/punctuation_capitalization_train.py>`__.
+
+    An example script on how to run evaluation and inference could be found here: `NeMo/examples/nlp/token_classification/punctuation_capitalization_evaluate.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/punctuation_capitalization_evaluate.py>`__.
+
+    The default configuration file for the model could be found at: `NeMo/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml>`__.
+
+
+
+.. _raw_data_format_punct:
+
+Raw Data Format
+---------------
+
+The Punctuation and Capitalization model can work with any text dataset, although it is recommended to balance the data, especially for the punctuation task.
+Before pre-processing the data to the format expected by the model, the data should be split into train.txt and dev.txt (and optionally test.txt).
+Each line in the **train.txt/dev.txt/test.txt** should represent one or more full and/or truncated sentences.
+
+Example of the train.txt/dev.txt file:
+
+.. code::
+
+    When is the next flight to New York?
+    The next flight is ...
+    ....
+
+
+The `source_data_dir` structure should look like this:
+
+.. code::
+
+   .
+   |--sourced_data_dir
+     |-- dev.txt
+     |-- train.txt
+
+
+
+NeMo Data Format for training the model
```

Comment:
"Data Format for Punc.. and Cap... Model" or just "Data Format" or "NeMo Data Format"?